x/txfees/module.go

```diff
@@ -175,48 +190,42 @@ func (AppModule) ConsensusVersion() uint64 { return 1 }
 // If they have, we unmarshal the current consensus params, update the target gas, and cache the value.
 // This is done to improve performance by not having to fetch and unmarshal the consensus params on every block.
 // TODO: Move this to EIP-1559 code
+// UNFORKING v2 TODO: Do we still want to use cachedConsParams here? I guess it removes the need to do arithmetic operations on every block.
```

Comment:
.

Reply:
I could get this to work again actually, I will need to make a change to the sdk but I think its worth it

Reply:
Actually disregard, it would require a larger refactor that would not be worth the delta.